build/nuke/Build.Native.cs

```diff
@@ -99,4 +99,73 @@ string AndroidHome
                 }
             )
     );
+
+    AbsolutePath GLFWPath => RootDirectory / "build" / "submodules" / "GLFW";
+    Target GLFW => CommonTarget
+    (
+        x => x.Before(Compile)
+            .Executes
+            (
+                () =>
+                {
+                    var @out = GLFWPath / "build";
+                    var prepare = "cmake -S. -B build -D BUILD_SHARED_LIBS=ON";
+                    var build = "cmake --build build --config Release";
+                    EnsureCleanDirectory(@out);
+                    var runtimes = RootDirectory / "src" / "Native" / "Silk.NET.GLFW.Native" / "runtimes";
+                    if (OperatingSystem.IsWindows())
+                    {
+                        InheritedShell($"{prepare} -A X64", GLFWPath)
+                            .AssertZeroExitCode();
+                        InheritedShell(build, GLFWPath)
+                            .AssertZeroExitCode();
+                        CopyAll(@out.GlobFiles("src/Release/glfw3.dll"), runtimes / "win-x64" / "native");
+                        
+                        EnsureCleanDirectory(@out);
+                        
+                        InheritedShell($"{prepare} -A Win32", GLFWPath)
+                            .AssertZeroExitCode();
+                        InheritedShell(build, GLFWPath)
+                            .AssertZeroExitCode();
+                        
+                        CopyAll(@out.GlobFiles("src/Release/glfw3.dll"), runtimes / "win-x86" / "native");
+                    }
+                    else if (OperatingSystem.IsLinux())
+                    {
+                        InheritedShell($"{prepare} -DCMAKE_SYSTEM_PROCESSOR=x86_64", GLFWPath)
+                            .AssertZeroExitCode();
+                        InheritedShell(build, GLFWPath)
+                            .AssertZeroExitCode();
+                        CopyAll(@out.GlobFiles("src/libglfw.so"), runtimes / "linux-x64" / "native");
+
+                        EnsureCleanDirectory(@out);
+                        
+                        InheritedShell($"{prepare} -DCMAKE_SYSTEM_PROCESSOR=i368", GLFWPath)
```

Comment:
`i368` doesn't sound right... `i386`?
```suggestion
                        InheritedShell($"{prepare} -DCMAKE_SYSTEM_PROCESSOR=i386", GLFWPath)
```

Reply:
Huh, surprised that it did not give any sort of warning/error?

Reply:
It's probably falling back to a sneaky default.